ath.sh

```diff
@@ -3,40 +3,40 @@ set -o errexit
 set -o nounset
 set -o pipefail
 set -o xtrace
-cd $(dirname $0)
+cd "$(dirname "$0")"
 
 # https://github.com/jenkinsci/acceptance-test-harness/releases
 export ATH_VERSION=5478.vb_b_cd04943676
 
 # TODO use Artifactory proxy?
 
-[ -f war/target/jenkins.war ] || mvn -B -ntp -Pquick-build -am -pl war package
+[[ -f war/target/jenkins.war ]] || mvn -B -ntp -Pquick-build -am -pl war package
```

Comment:
[Articles like this](https://stackoverflow.com/questions/669452/are-double-square-brackets-preferable-over-single-square-brackets-in-b) point out that if you're going to be using `bash(1)` anyway, using its constructs like `[[` over `[` are generally safer and have fewer surprises.